service-api/app/test/AppTest/Service/Lpa/OlderLpaServiceTest.php

```diff
@@ -1066,9 +1075,58 @@ public function allow_user_to_continue_request_for_activation_key_if_lpa_match_f
                 )
             );
 
-       $response = $service->validateOlderLpaRequest($this->userId, $dataToMatch);
-       $this->assertIsArray($response);
+        $response = $service->validateOlderLpaRequest($this->userId, $dataToMatch);
+        $this->assertIsArray($response);
+
+        $this->assertEquals($lpaMatchResponse, $response);
+    }
+
+    /**
+     * @test
+     * @throws Exception
+     */
+    public function older_lpa_lookup_throws_an_exception_if_lpa_registration_status_not_valid()
+    {
+        $lpa = new Lpa(
+            [
+                'uId' => $this->lpaUid,
+                'registrationDate' => '2019-10-31',
+                'status' => 'Pending',
+            ],
+            new DateTime()
+        );
+
+        $dataToMatch = [
+            'reference_number' => $this->lpaUid,
+            'dob' => '1980-03-01',
+            'first_names' => 'Test Tester',
+            'last_name' => 'Testing',
+            'postcode' => 'Ab1 2Cd',
+            'force_activation_key' => false,
+        ];
+
+        $service = $this->getOlderLpaService();
 
-       $this->assertEquals($lpaMatchResponse, $response);
+        $this->lpaAlreadyAddedProphecy
+            ->__invoke($this->userId, $this->lpaUid)
+            ->willReturn(null);
+
+        $this->lpaServiceProphecy
+            ->getByUid($this->lpaUid)
+            ->willReturn($lpa);
+
+        $expectedException = new NotFoundException(
+            'LPA status invalid',
+        );
+
+        $this->validateOlderLpaRequirementsProphecy
+            ->__invoke($lpa->getData())
+            ->willThrow($expectedException);
```

Comment:
```suggestion
            ->willThrow(new NotFoundException('LPA status invalid'));
```
as you're not using the $expectedException anywhere else, its probably just best to do it this way